### PR TITLE
Check response status when calling MMN

### PR DESCRIPTION
**Pull request by tomasmik**
- Close body
- Check response code

### CODE REVIEW

File: mmn/client.go [hunk 1]
```diff
@@ -91,10 +91,17 @@ func (m *client) UpdateBeneficiary(data *UpdateBeneficiaryRequest) error {
...
 		return err
 	}
+	defer resp.Body.Close()
+
 	body, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
 		return err
 	}
+
+	if resp.StatusCode < 199 || resp.StatusCode > 299 {
```
Comment (Snawoot) [inline mmn/client.go:101]:
```suggestion
	if resp.StatusCode < 200 || resp.StatusCode > 299 {
```

Comment (Snawoot) [reply mmn/client.go:101]: Don't want 199 to be considered OK status

File: mmn/client.go [hunk 2]
```diff
@@ -121,11 +128,17 @@ func (m *client) GetBeneficiary(identityStr string) (string, error) {
...
 		return "", err
 	}
+	defer resp.Body.Close()
+
 	body, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
 		return "", err
 	}
 
+	if resp.StatusCode < 199 || resp.StatusCode > 299 {
```
Comment (Snawoot) [inline mmn/client.go:138]:
```suggestion
	if resp.StatusCode < 200 || resp.StatusCode > 299 {
```

Same

**Review by Snawoot — APPROVED** [review]

File: mmn/client.go (hunk 1 above)
Comment (tomasmik) [reply mmn/client.go:101]: oops

**Review by soffokl — APPROVED** [review]

**Pull request merged**